usecases/objects/references.go

```diff
@@ -35,8 +35,7 @@ func (m *Manager) autodetectToClass(ctx context.Context, principal *models.Princ
 	}
 
 	toClass := prop.DataType[0] // datatype is the name of the class that is referenced
-	beaconElements := strings.Split(string(beaconRef), "/")
-	toUUID := beaconElements[len(beaconElements)-1]
-	toBeacon := "weaviate://localhost/" + string(toClass) + "/" + toUUID // datatype is the name of the class that is referenced
+	toBeacon := crossref.New("localhost", toClass, beaconRef.TargetID).String()
```

Comment:
there is also `crossref.NewLocalhost(...)` constructor

Reply:
done, thx